source/renderer/app/components/wallet/settings/paper-wallet-create-certificate-dialogs/PaperWalletCreateCertificateCompletionDialog.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import classnames from 'classnames';
+import { defineMessages, intlShape } from 'react-intl';
+import Checkbox from 'react-polymorph/lib/components/Checkbox';
+import SimpleCheckboxSkin from 'react-polymorph/lib/skins/simple/raw/CheckboxSkin';
+import Dialog from '../../../widgets/Dialog';
+import DialogCloseButton from '../../../widgets/DialogCloseButton';
+import DialogBackButton from '../../../widgets/DialogBackButton';
+import globalMessages from '../../../../i18n/global-messages';
+
+import styles from './PaperWalletCreateCertificateCompletionDialog.scss';
+
+const messages = defineMessages({
+  headline: {
+    id: 'paper.wallet.create.certificate.completion.dialog.headline',
+    defaultMessage: '!!!Paper wallet certificate',
+    description: 'Headline for the "Paper wallet create certificate completion dialog" headline.'
+  },
+  subtitle: {
+    id: 'paper.wallet.create.certificate.completion.dialog.subtitle',
+    defaultMessage: '!!!Now you can fold your paper wallet certificate and glue together all the parts. Keep certificate safe. To import wallet back crop glued certificate’s edges to reach inner part.',
+    description: 'Headline for the "Paper wallet create certificate completion dialog" subtitle.'
+  },
+  linkInstructions: {
+    id: 'paper.wallet.create.certificate.completion.dialog.linkInstructions',
+    defaultMessage: '!!!You can use this link to opent the address in Cardano Explorer, save it in your browser bookmarks and share it with others to receive funds:',
```

Comment:
There is a small typo here

Reply:
Indeed you are right - it should be `open` in place of `opent`. Thanks for the heads up! ;)
cc @tomothespian